__tests__/app.test.js

```diff
@@ -36,16 +48,57 @@ describe("GET /api", () => {
       .get("/api")
       .expect(200)
       .then((response) => {
-        expect(typeof response.text).toBe("string");
+        expect(typeof response.body).toBe("object");
       });
   });
   test("should check that the file return from the endpoint is equal to the endpoints.json required in to the test suite", () => {
     return request(app)
       .get("/api")
       .expect(200)
       .then((response) => {
-        const parsedApiInfo = JSON.stringify(apiInformation, null, 2);
-        expect(response.text).toEqual(parsedApiInfo);
+        expect(response.body).toEqual(apiInformation);
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("should respond with 200 status code", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+
+  test("should return a specific article based on its id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const article = response.body;
```

Comment:
Good start. However, this test needs to test for the specific article_id - otherwise how do you know you've got the right one back?